### PR TITLE
Add support for pandoc

### PR DESCRIPTION
**Pull request by tpoisot**
The current version of `nord-vim` does not have support for most syntax elements from the `vim-pandoc-syntax` package (#207):

before

I have added support for these groups - notably, this solves the issues with `Conceal` (notably #211). This is the result with the additional syntax groups:

after

### CODE REVIEW

**Comment by BirgerNi** [comment]
Hej @tpoisot,

just tested your pull request, good job. Do you use an option like `hi Conceal guibg=NONE ctermbg=NONE` in your `.vimrc`?

1) Pull request w/o conceal option in `.vimrc`


2) Pull request w/ conceal option in `.vimrc`


3) develop branch w/ conceal option in `.vimrc`

**Comment by tpoisot** [comment]
You're right, I did have that! Should it be added to the theme? I'm not sure how it can interact with other packages.

**Comment by BirgerNi** [comment]
I'm okay with that. I guess at least some documentation would be nice for new users. Any thoughts @arcticicestudio ?

**Comment by lompabo** [comment]
Anyone for reviewing this? Pandoc support is currently the main factor preventing me from switching to this theme

File: colors/nord.vim [hunk 1]
```diff
@@ -677,6 +677,19 @@ hi! link jsNoise Delimiter
...
+call s:hi("pandocPipeTableDelims", s:nord3_gui, "", s:nord3_term, "", "", "")
+call s:hi("pandocPipeTableHeader", s:nord3_gui, "", s:nord3_term, "", "", "")
+call s:hi("pandocTableHeaderWord", s:nord9_gui, "", s:nord9_term, "", "", "")
+call s:hi("pandocCiteAnchor", s:nord12_gui, "", s:nord12_term, "", "", "")
+call s:hi("pandocCiteKey", s:nord15_gui, "", s:nord15_term, "", "", "")
+call s:hi("pandocReferenceDefinition", s:nord6_gui, "", s:nord6_term, "", "", "")
+call s:hi("pandocFootnoteDef", s:nord9_gui, "", s:nord9_term, "", "", "")
+call s:hi("pandocDefinitionBlockTerm", s:nord12_gui, "", s:nord12_term, "", s:bold, "")
+call s:hi("pandocDefinitionBlock", s:nord9_gui, "", s:nord9_term, "", s:italic, "")
+
```
Comment (arcticicestudio) [inline colors/nord.vim:692]: I think linking most groups to existing Markdown groups should ensure to get a consistent style across languages and different plugins.

```suggestion
" Pandoc
" > vim-pandoc/vim-pandoc-syntax
call s:hi("pandocDefinitionBlockTerm", s:nord7_gui, "", s:nord7_term, "", s:italic, "")
call s:hi("pandocTableDelims", s:nord3_gui, "", s:nord3_term, "", "", "")
hi! link pandocAtxHeader markdownH1
hi! link pandocBlockQuote markdownBlockquote
hi! link pandocCiteAnchor Operator
hi! link pandocCiteKey pandocReferenceLabel
hi! link pandocDefinitionBlockMark Operator
hi! link pandocEmphasis markdownItalic
hi! link pandocFootnoteID pandocReferenceLabel
hi! link pandocFootnoteIDHead markdownLinkDelimiter
hi! link pandocFootnoteIDTail pandocFootnoteIDHead
hi! link pandocGridTableDelims pandocTableDelims
hi! link pandocGridTableHeader pandocTableDelims
hi! link pandocOperator Operator
hi! link pandocPipeTableDelims pandocTableDelims
hi! link pandocReferenceDefinition pandocReferenceLabel
hi! link pandocReferenceLabel markdownLinkText
hi! link pandocReferenceURL markdownUrl
hi! link pandocSimpleTableHeader pandocAtxHeader
hi! link pandocStrong markdownBold
hi! link pandocTableHeaderWord pandocAtxHeader
hi! link pandocUListItemBullet Operator

```

**Review by arcticicestudio — CHANGES_REQUESTED** [review]
Hi @tpoisot :wave:, thanks for your contribution :+1:

Thanks for your patience, the large amount of tasks for the Nord project are really time consuming and free time is kind of rare.
I've suggested some changes to make it more consistent with existing theme styles. Let me know what you think about it.

**Comment by tpoisot** [comment]
@arcticicestudio - will have a look, thanks for the feedback!

**Comment by arcticicestudio** [comment]
@tpoisot Let me know if you'd like to apply the suggestions so I can resolve the current merge conflicts.

**Comment by tpoisot** [comment]
I would love to, hopefully I'll find the time next week

**Comment by tpoisot** [comment]
I think the merge conflicts are gone!

**Review by svengreb — APPROVED** [review]

**Review by arcticicestudio — APPROVED** [review]
LGTM 👍🏼 Thanks again for your contribution.

**Pull request merged**